config/website.conf

```diff
@@ -0,0 +1,60 @@
+server {
+    listen 443 ssl http2 default_server;
+
+    server_name www.ch.tudelft.nl ch.ewi.tudelft.nl ch.its.tudelft.nl;
+
+    include /etc/nginx/ssl/ssl.conf;
+    ssl_certificate /etc/letsencrypt/live/ch.tudelft.nl/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/ch.tudelft.nl/privkey.pem;
+
+    include /etc/nginx/shared/*.conf;
+
+    location / {
+        add_header Strict-Transport-Security "max-age=15768000; includeSubDomains; preload" always;
+
+        return 302 https://ch.tudelft.nl/;
+    }
+}
+
+server {
+    listen 443 ssl http2;
+
+    server_name ch.tudelft.nl;
+
+    include /etc/nginx/ssl/ssl.conf;
+    ssl_certificate /etc/letsencrypt/live/ch.tudelft.nl/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/ch.tudelft.nl/privkey.pem;
+
+    include /etc/nginx/ssl/proxy.conf;
+    include /etc/nginx/shared/*.conf;
+
+    client_max_body_size 32M;
+
+    location /wiki {
+        proxy_pass https://frans.chnet/wiki;
+    }
+
+    location /password {
+        proxy_pass https://password.k8s.chnet:32042/password;
+    }
+
+    location /events {
+        proxy_pass https://events.k8s.chnet/events;
+    }
+
+    location /payments {
+        proxy_pass https://payments.k8s.chnet/payments;
+    }
+
+    location /choice2 {
+        proxy_pass https://choice.w3cie.k8s.chnet/choice2;
+    }
+
+    location / {
+        add_header Strict-Transport-Security "max-age=15768000; includeSubDomains; preload" always;
```

Comment:
HSTS is already added by the Kubernetes ingress controller

Reply:
So it is not needed here?

Reply:
Nope

Reply:
✅